### DISCUSSION TITLE
Ubuntu

### DISCUSSION THREAD

**Discussion by hoffmann04** [JojiiOfficial/LiveBudsCli#46] [Q&A]
Hey, thanks for this code, its really awesome the support for linux! I recently buy a Buds+ and I use Ubuntu in my daily basis for my job. I want to know how can I install this in Ubuntu? Thanks!

**Comment by JojiiOfficial** [comment]
Install cargo and rust (eg. via rustup) and follow the [compilation section](https://github.com/JojiiOfficial/LiveBudsCli#compilation) on the readme. I didn't package it for debian based systems since I don't use them.

**Reply by hoffmann04** [reply]
Thanks for the reply! I will try in my spare time.
